test/powershell/Language/Scripting/I18n.Tests.ps1

```diff
@@ -0,0 +1,178 @@
+#This works only for en-US or fr-FR
+if ($PSUICulture -ne 'en-US' -and $PSUICulture -ne 'fr-FR')
+{
+    return
+}
```

Comment:
There is an issue with writing tests with this pattern, where we won't really know what tests we would have run if the conditions were met. Could you rewrite these as follows?
Note, that if the condition isn't met all the tests will be skipped, but we'll report them as skipped.

Sometimes you can't avoid the `return` at the top of the file, but this isn't one of those cases


```
Describe 'Testing of script internationalization' -Tags "CI" {
    BeforeAll {
        $dir=$PSScriptRoot
        $defaultParamValues = $PSDefaultParameterValues.Clone()
        #This works only for en-US or fr-FR
        if ($PSUICulture -ne 'en-US' -and $PSUICulture -ne 'fr-FR')
        {
            $PSDefaultParameterValues["It:Skip"] = $true
        }
    }
    AfterAll {
        $global:PSDefaultParameterValues = $defaultParamValues
    }
    

    It 'convertFromString-Data should work with data statement.' {

        data mydata
        {
            ConvertFrom-StringData @'
        string1=string1
        string2=string2
'@
        }

        $mydata.string1 | Should Be 'string1'
        $mydata.string2 | Should Be 'string2' 
    }

    It 'Import default culture data is done correctly' {

        import-localizedData mydata;

        $mydata.string1 | Should Be 'string1 for en-US'
        $mydata.string2 | Should be 'string2 for en-US'
    }

    It 'Import data with specific culture (en-US)' {

        import-localizedData mydata -uiculture en-US

        $mydata.string1 | Should Be 'string1 for en-US'
        $mydata.string2 | Should Be 'string2 for en-US'
    }
```